docs/ir/ir-breaking-changes.rst

```diff
@@ -4,3 +4,34 @@ Solidity IR-based Codegen Changes
 
 This section highlights the main differences between the old and the IR-based codegen,
 along with the reasoning behind the changes and how to update affected code.
+
+Semantic Only Changes
+=====================
+
+This section lists the changes that are semantic-only, thus potentially
+hiding new and different behavior in existing code.
+
+ * When deleting storage struct, if struct occupies less than one slot, whole slot will be cleared.
```

Comment:
Needs some copy editing.

Reply:
I changes this a bit, hope it looks better now